core/src/main/scala/org/apache/spark/ui/jobs/JobProgressListener.scala

```diff
@@ -109,7 +111,8 @@ class JobProgressListener(conf: SparkConf) extends SparkListener with Logging {
       "failedJobs" -> failedJobs.size,
       "completedStages" -> completedStages.size,
       "skippedStages" -> skippedStages.size,
-      "failedStages" -> failedStages.size
+      "failedStages" -> failedStages.size,
+      "jobGroupToJobIds" -> jobGroupToJobIds.values.map(_.size).sum
```

Comment:
IIUC, based on when elements are removed, the size of this should be the same as the size of `jobIdToData`.  Why not okace `jobGroupToJobIds` alongside it in `getSizesOfSoftSizeLimitedCollections`?